src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-91000.mdx

```diff
@@ -0,0 +1,43 @@
+---
+subject: Python agent
+releaseDate: '2024-05-23'
+version: 9.10.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Add support for AIOBotocore', 'Add support for Meta Llama3', 'Add support for Mistral AI']
+bugs: ['Fix OpenAI error parsing', 'Record loguru timing in milliseconds']
+security: []
+---
+
+## Notes
+
+This release of the Python agent adds instrumentation for [AIOBotocore](https://pypi.org/project/aiobotocore/), support for Meta Llama3 and Mistral AI in Amazon Bedrock, fixes an error parsing issue in [OpenAI](https://pypi.org/project/openai/), and updates [Loguru](https://pypi.org/project/loguru/)'s instrumentation to use milliseconds instead of seconds.
```

Comment:
```suggestion
This release of the Python agent:
* Adds instrumentation for [AIOBotocore](https://pypi.org/project/aiobotocore/)
* Adds support for Meta Llama3 and Mistral AI in Amazon Bedrock
* Fixes an error parsing issue in [OpenAI](https://pypi.org/project/openai/)
* Updates [Loguru](https://pypi.org/project/loguru/)'s instrumentation to use milliseconds instead of seconds
```

Reply:
Sounds good!